### PR TITLE
Replace ugettext and smart_text with proper functions for Django 4 compatibility

### PR DESCRIPTION
**Pull request by MichelML**
see https://github.com/labd/django-cognito-jwt/issues/39

### CODE REVIEW

**Comment by MichelML** [comment]
@mvantellingen @mikedebock can you please check and merge, and publish a new patch or minor when this is done please

without this it's impossible to upgrade to django-4

**Comment by MichelML** [comment]
@mikedebock do you have the rights to merge this? You seem more active compared to the initial maintainer @mvantellingen

**Comment by MichelML** [comment]
hello @mikedebock , sorry to push for that, this is a minor PR and I can see you are active on GH - do you guys plan on making any additional change/maintenance on this package or not?

**Comment by MichelML** [comment]
good morning @mikedebock @mvantellingen , it would just be nice to have an answer from you guys.

Thanks in advance :pray:

File: src/django_cognito_jwt/backend.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 from django.apps import apps as django_apps
 from django.conf import settings
-from django.utils.encoding import smart_text
```
Comment (mikedebock) [inline src/django_cognito_jwt/backend.py:5]: At line 41 `smart_text` is still used, can you replace that one for `force_str`?

**Comment by mikedebock** [comment]
Hi @MichelML,

Thank you for your pull request. I've a small finding, would you process my small feedback?
After that I'll merge this PR en check out if i'm able to create a new release for you!

**Comment by MichelML** [comment]
Thanks @mikedebock , this is fixed :+1: please let me know of any updates

**Comment by MichelML** [comment]
@mikedebock seems unclear to me at the moment wether the failing UTs are because of this change https://github.com/labd/django-cognito-jwt/runs/4486077441?check_suite_focus=true

there seems to be a formatting issue also, sorry for that

**Comment by mikedebock** [comment]
@MichelML I've fixed the formatting error, but the build is still failing. I think because of outdated (not-pinned) dependencies.
Can you take care of succeeding the build? You can check this locally by running: `pip install tax && tox -e py38-django30`

**Comment by MichelML** [comment]
looking into it

**Comment by MichelML** [comment]
@mikedebock you can check this PR which is ready to be merged I think https://github.com/labd/django-cognito-jwt/pull/41

**Comment by mikedebock** [comment]
@MichelML looks like the credentials of pypi are invalid, I'm on int!

**Pull request merged**